src/plugins.d/pluginsd_parser.c

```diff
@@ -235,6 +235,7 @@ static inline PARSER_RC pluginsd_host_define_end(char **words __maybe_unused, si
         host->rrdlabels = parser->user.host_define.rrdlabels;
         parser->user.host_define.rrdlabels = NULL;
     }
+    rrdlabels_add(host->rrdlabels, "_is_virtual_node", "true", RRDLABEL_SRC_AUTO);
```

Comment:
Do we need to set it to false for regular nodes?

Reply:
Reverting this change (`_is_virtual_node`) -- will handle in another PR to adjust streaming.